src/main/webapp/assets/js/index.js

```diff
@@ -63,12 +63,15 @@ function initMap(foodLocations) {
 /**
  * Returns a product listing on the home page.
  */
-function homePageListingTemplate(product) {
-  const baseImage = product.photoBlobstoreUrlList.length === 0 ? "" : product.photoBlobstoreUrlList[0];
+function homePageListingTemplate(business) {
+  const product = business.data;
+  const baseImage = product.logoBlobstoreUrl;
   const description = truncateWords(product.description, INITIAL_DESC_WORDS)
   return `<div class="product-listing-card">
             <div class='product-listing-image'>
-            <img src=${baseImage}>
+            <a href="/product.html?businessID=${business.id}">
```

Comment:
template literals require backticks, not quotation marks
```
`/product.html?businessID=${business.id}`
```

Reply:
It worked for me because the entire return statement is backticked. But I will convert to an element and use appendChild!